### PR TITLE
revert mnemonic in settings

### PR DESCRIPTION
**Pull request by flexsurfer**
revert mnemonic in settings

### CODE REVIEW

**Comment by flexsurfer** [comment]
this didn't help, still no mnemonic

**Comment by qfrank** [comment]
image
actually it's emptyed here @flexsurfer

File: api/geth_backend.go [hunk 1]
```diff
@@ -1456,15 +1456,11 @@ func (b *GethStatusBackend) prepareSettings(info generator.GeneratedAccountInfo,
...
 	settings.TestNetworksEnabled = request.TestNetworksEnabled
-
-	// If restoring an account, we don't set the mnemonic
-	if info.Mnemonic == "" {
-		settings.Mnemonic = &info.Mnemonic
-		settings.OmitTransfersHistoryScan = true
-		// TODO(rasom): uncomment it as soon as address will be properly
-		// marked as shown on mobile client
-		//settings.MnemonicWasNotShown = true
-	}
```
Comment (qfrank) [inline api/geth_backend.go:1467]: Sorry I made a wrong guess 🥲

**Comment by flexsurfer** [comment]
i'll leave it for you @qfrank and @IvanBelyakoff , I'm not familiar with that code

**Comment by qfrank** [comment]
Okay, I'll fix it @IvanBelyakoff

**Pull request closed without merging**